pom.xml

```diff
@@ -79,21 +81,39 @@
   </distributionManagement>
 
   <properties>
-    <checkstyle.version>6.1.1</checkstyle.version>
+    <maven.compiler.source>1.6</maven.compiler.source>
+    <maven.compiler.target>1.6</maven.compiler.target>
+
+    <version.com.puppycrawl.tools.checkstyle>6.1.1</version.com.puppycrawl.tools.checkstyle>
+    <version.com.mycila.license-maven-plugin>2.7</version.com.mycila.license-maven-plugin>
+    <version.org.eclipse.jgit>3.6.1.201501031845-r</version.org.eclipse.jgit>
+    <version.junit>4.12</version.junit>
   </properties>
```

Comment:
Because that's how it is done - not only usually but also in the Hawkular family. It is practical to do things in uniform ways so that one can find them quickly in a well defined location. Moreover, doing it the same way in the whole family of hawkular projects will make it easier to keep the same versions on multiple place. As far as I can see now, it will be inevitable to copy e.g. `com.puppycrawl.tools.checkstyle` version between here and the `hawkular` ur-parent.
You are right that there is no re-use in this particular place but for the named reasons, I'd keep it like this.